Simulation/TSoG_X1_Sim.m

```diff
@@ -0,0 +1,95 @@
+% Based off of https://github.com/TSoGiants/MECH2103-Computational-Aerodynamics
+%  TSoG_X1_Sim is the main function to run the X1 simulation. It can be run with
+## or without an input Test Case (the sim will run with default inputs if no Test
```

Comment:
I'm not sure what kind of comment `##` is but we should probably stick to `%` everywhere for consistency.

Reply:
I set up my editor to do wrap around on some designated line column number (80) and that's what the editor shows up when it does the wraparound.

I'll just manually change it to %, I didn't really like that it did that even if it makes it working easier.

Reply:
Makes sense. It certainly is pretty weird and confusing that there are many ways to create a comment but maybe `##` is more idiomatic than this one or something like that. I have no preference but since we started using `%` we should probably just keep using it. 
There also probably is an option to change the comment style your editor uses.

Reply:
Fixed the formatting to match the  rest of the code